packages/bun-types/globals.d.ts

```diff
@@ -75,6 +75,34 @@ interface ArrayConstructor {
   ): Promise<Array<T>>;
 }
 
+type UncaughtExceptionOrigin = "uncaughtException" | "unhandledRejection";
```

Comment:
We have not implemented `unhandledRejection`.

Reply:
This isn't actually used. When we implement it we can just uncomment this line further down

```ts
// on(event: "uncaughtException", listener: UncaughtExceptionListener): this;
```